Add new integration for AiDot (#37320)
home-assistant/home-assistant.github.io: source/_integrations/aidot.markdown

---
title: AiDot
description: Instructions on integrating AiDot home security with Home Assistant.
ha_category:
  - Light
ha_release: 2024.6
ha_iot_class: cloud_polling
ha_config_flow: true
ha_codeowners:
  - '@s1eedz'
ha_domain: aidot
ha_homekit: true
ha_platforms:
  - light
ha_integration_type: integration
---

> The [AiDot](https://www.aidot.com/) {% term integration %} allows you to control the Wi-Fi lights with Home Assistant.

Reviewer: ```suggestion
The **AiDot** {% term integration %} allows you to control your [AiDot](https://www.aidot.com/) Wi-Fi lights with Home Assistant.
```

## Prerequisites

> - To be able to configure your machine in Home Assistant, your machine needs to be added to your account using the official AiDot app first.

Reviewer: which machine? the AiDot Wi-Fi light?

- Only login with username & password is supported.
{% include integrations/config_flow.md %}

{% configuration_basic %}
Country:
  description: "Your country you use to log into the AiDot app."
Username:
  description: "Your username you use to log into the AiDot app."
Password:
  description: "Password you use to log into the AiDot app."
{% endconfiguration_basic %}

## Data updates

> - After successful login, the device list will be pulled from AiDot Cloud first

Reviewer: ```suggestion
- After successful login, the device list will be {% term polling polled %} from AiDot Cloud first
```
Do you mean "polling"?
We have a [glossary term reference](https://developers.home-assistant.io/docs/documenting/integration-docs-examples#glossary-term-reference) for this.

> - After pulling the list, the integration will pull the latest list from the cloud every 6 hours and process additions and deletions

Reviewer: ```suggestion
- After {% term polling %} the list, the integration will poll the latest list from the cloud every 6 hours and process additions and deletions
```

- The integration will update its lights by polling the AiDot Bridge every five seconds for new values.

## Supported devices

> - A19

Reviewer: Is this a product line? Or a device brand name? Is the full name something like AiDot Linkind A19 Smart WiFi Light Bulbs?

- BR30

## Troubleshooting

There are no commonly known issues with this integration.

> ## Remove integration

Reviewer: ```suggestion
## Removing the integration
```

---

> title: Aidot

Reviewer 2: ```suggestion
title: AiDot
```

> description: Instructions on integrating Aiot home security with Home Assistant.

Reviewer 2: is it Aiot or AiDot?

ha_category:
  - Light
ha_release: 2024.6

> ha_iot_class: local_polling

Reviewer 2: from the description it should be "Cloud Polling"

ha_config_flow: true
ha_codeowners:
  - '@s1eedz'

> ha_domain: Aidot

Reviewer 2: ```suggestion
ha_domain: aidot
```

ha_homekit: true
ha_platforms:
  - light
ha_integration_type: integration
---

> The [Aidot](https://www.aidot.com/) {% term integration %} allows you to control the Wi-Fi lights with Home Assistant.

Reviewer 2: ```suggestion
The [AiDot](https://www.aidot.com/) {% term integration %} allows you to control the Wi-Fi lights with Home Assistant.
```

> {% include integrations/config_flow.md %}

Reviewer 2: the instructions are already in the Prerequisites section, so they can be removed here to avoid duplicates

## Prerequisites
- To be able to configure your machine in Home Assistant, your machine needs to be added to your account using the official AiDot app first.
- Only login with username & password is supported. 
{% include integrations/config_flow.md %}

{% configuration_basic %}
Country:
  description: "Your country you use to log into the AiDot app."
Username:
  description: "Your username you use to log into the AiDot app."
Password:
  description: "Password you use to log into the AiDot app."
{% endconfiguration_basic %}

## Data updates
- After successful login, the device list will be pulled from AiDot Cloud first
- After pulling the list, the integration will pull the latest list from the cloud every 6 hours and process additions and deletions
- The integration will update its lights by polling the AiDot Bridge every one seconds for new values.

## Supported devices

- A19
- BR30

## Troubleshooting

There are no commonly known issues with this integration.

## Remove integration

> This integration follows standard integration removal. No extra steps are required.
> 

Reviewer 2: ```suggestion
```

should not be needed, since the include already explains the complete removal
